NumLib/Extrapolation/ExtrapolatableElementCollection.h

```diff
@@ -0,0 +1,121 @@
+/**
+ * \copyright
+ * Copyright (c) 2012-2016, OpenGeoSys Community (http://www.opengeosys.org)
+ *            Distributed under a Modified BSD License.
+ *              See accompanying file LICENSE.txt or
+ *              http://www.opengeosys.org/project/license
+ *
+ */
+
+#ifndef NUMLIB_EXTRAPOLATION_EXTRAPOLATABLEELEMENTCOLLECTION_H
+#define NUMLIB_EXTRAPOLATION_EXTRAPOLATABLEELEMENTCOLLECTION_H
+
+#include "ExtrapolatableElement.h"
+
+namespace NumLib
+{
+//! Adaptor to get information needed by an Extrapolator from an "arbitrary"
+//! collection of elements (e.g., local assemblers).
+class ExtrapolatableElementCollection
```

Comment:
can you rename this class or add document to clarify this class is indeed a base class?